examples/trials/mnist-hyperband/mnist.py

```diff
@@ -207,38 +208,31 @@ def main(params):
         logger.debug('Final result is %g', test_acc)
         logger.debug('Send final result done.')
 
-
-def generate_default_params():
-    '''
-    Generate default parameters for mnist network.
-    '''
-    params = {
-        'data_dir': '/tmp/tensorflow/mnist/input_data',
-        'dropout_rate': 0.5,
-        'channel_1_num': 32,
-        'channel_2_num': 64,
-        'conv_size': 5,
-        'pool_size': 2,
-        'hidden_size': 1024,
-        'learning_rate': 1e-4,
-        'batch_size': 32}
-    return params
-
+def get_params():
+    ''' Get parameters from command line '''
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--data_dir", type=str, default='/tmp/tensorflow/mnist/input_data', help="data directory")
```

Comment:
I don't think we need to update like argparse. Because you only use the default value.

Reply:
This is just to keep the other parts consistent with the MNIST example, then users can find the difference between the trial code examples(to know where and how to control the budget).

Reply:
I think it's easy code to understand and we don't have to change this part...

Reply:
In addition to this non-uniformity, the unification here is mainly to **change the batch_num 10 times to 100 times**, otherwise the result of Hyperband example's is always lower than TPE/SMAC.

Reply:
Different training codes can cause a lot of misunderstandings about Tuner's performance. I think this part of the version may be better than the original? (although it doesn't matter)

Reply:
If you want to change, could you consider to change all the mnist in examples?

Reply:
Will start a new pr to do this.